packages/babel-plugin/src/utils/evaluate-path.js

```diff
@@ -173,9 +173,11 @@ function evaluateThemeRef(
         : utils.genFileBasedIdentifier({ fileName, exportName, key });
 
     const debug = state.traversalState.options.debug;
+    const enableDebugClassNames =
+      state.traversalState.options.enableDebugClassNames;
 
     const varName =
-      debug === true
+      debug && enableDebugClassNames
```

Comment:
I again think that `enableDebugClassNames` should default to the value of `debug` so we don't have to be careful about reading both values in all places that use it...